crates/sui-indexer-alt-graphql/src/api/types/transaction_effects.rs

```diff
@@ -193,13 +193,14 @@ impl EffectsContents {
         let transaction_digest = content.digest()?;
         let timestamp_ms = content.timestamp_ms();
 
-        for edge in cursors.edges {
+        for (idx, edge) in cursors.edges.iter().enumerate() {
             let event = Event {
                 scope: self.scope.clone(),
                 native: events[*edge.cursor].clone(),
                 transaction_digest,
                 sequence_number: *edge.cursor as u64,
                 timestamp_ms,
+                tx_sequence_number: idx as u64,
```

Comment:
This is not correct -- the index into `cursors.edges` is not the transaction sequence number for this transaction.